addenda14_test.go

```diff
@@ -78,14 +78,10 @@ func BenchmarkAddenda14Parse(b *testing.B) {
 func testAddenda14ValidRecordType(t testing.TB) {
 	addenda14 := mockAddenda14()
 	addenda14.recordType = "63"
-	if err := addenda14.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda14.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
return the fieldError	

if addenda14.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda14.recordType)
	}